src/metabase/server/middleware/log.clj

```diff
@@ -195,11 +196,26 @@
 ;; Actual middleware. Determines whether request should be logged, and, if so, creates the info dictionary and hands
 ;; off to functions above.
 
+(defsetting health-check-logging-enabled
+  (deferred-tru "Whether to log health check requests from session middleware.")
+  :type       :boolean
+  :default    true
+  :visibility :internal
+  :export?    false)
+
+(defn- logging-disabled-uris
```

Comment:
This feels a bit complicated, filtering, vector concatenation, seq-ing, removing, and then turning into a set. What about these alternative implementations?

```clojure
(defn- logging-disabled-uris
  "The set of URIs that should not be logged."
  []
  (cond-> #{"/api/util/logs"}
    (not (health-check-logging-enabled) (conj "/api/health"))))

(let [base #{"/api/util/logs"}
      with-health (conj base "/api/health")]
  (defn- logging-disabled-uris
    "The set of URIs that should not be logged."
    []
    (if (health-check-logging-enabled)
      base
      with-health)))
```

Reply:
Good call, got a bit of tunnel vision trying to adapt the existing approach